extensions/imagesvg/src/common/Types.ts

```diff
@@ -12,16 +12,40 @@ export interface ImageSvgStyle extends RXTypes.ViewStyle {
 }
 export declare type ImageSvgStyleRuleSet = RXTypes.StyleRuleSet<ImageSvgStyle>;
 
-export interface SvgCommonProps {
-    key?: string | number;
+type NumberProp = string | number;
```

Comment:
Why are you supporting strings here? No where else in ReactXP are strings supported for numeric props. I'd prefer to stay consistent.

Reply:
Because both the web and the native implementations support number|string, and it's much faster than having to do Number(x) everywhere.